### PR TITLE
support Store NSArray and NSDictionary

### PR DESCRIPTION
**Pull request by kingslay**

### CODE REVIEW

File: Pod/Classes/DatabaseConnection.swift [hunk 1]
```diff
@@ -12,6 +12,8 @@ import sqlite3
 /** Valid SQLite types are marked using the 'SQLiteValue' protocol */
 public protocol SQLiteValue {}
 
+extension NSArray: SQLiteValue {}
+extension NSDictionary: SQLiteValue {}
```
Comment (Oyvindkg) [inline Pod/Classes/DatabaseConnection.swift:16]: Do you think these should be defined in `SwiftyDB` instead? `TinySQLite` should not accept collections as it does not know how to handle store. It should probably be converted to NSData in `SwiftyDB` before being passed on

Comment (kingslay) [inline Pod/Classes/DatabaseConnection.swift:16]: i have try to defined 

`extension NSArray: SQLiteValue {}`

`extension NSDictionary: SQLiteValue {}`

in PropertyData.swift. but error

`PropertyData.swift:10:20: Redundant conformance of 'NSArray' to protocol 'SQLiteValue'`

`PropertyData.swift:11:25: Redundant conformance of 'NSDictionary' to protocol 'SQLiteValue'`

You can try it

Comment (Oyvindkg) [inline Pod/Classes/DatabaseConnection.swift:16]: Yes, i know. Maybe SwiftyDB should define its own protocol for valid value types. Would make it less dependent on TinySQLite, and make future changes easier.

Great pull request, by the way!

**Comment by Oyvindkg** [comment]
Protocol updates moved to SwiftyDB

**Pull request closed without merging**